pyproject.toml

```diff
@@ -17,21 +17,17 @@ include = [
 "nautobot" = "nautobot_chatops.workers.nautobot:nautobot"
 
 [tool.poetry.dependencies]
-python = "^3.6"
-nautobot = "*"
+python = "^3.7"
+nautobot = "^1.1.0"
```

Comment:
I thought this was pattern was against plugin policy as it can automatically upgrade a Nautobot instance.  That said, probably a safe value.

Reply:
Testing, this doesn't seem to break my nautobot instance. I am good with this change.